Code/Framework/AzToolsFramework/Tests/ActionManager/ActionManagerTests.cpp

```diff
@@ -14,59 +14,64 @@ namespace UnitTest
     TEST_F(ActionManagerFixture, RegisterActionContextWithoutWidget)
     {
         auto outcome =
-            m_actionManagerInterface->RegisterActionContext(nullptr, "o3de.context.editor.test", "Test", "");
+            m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, nullptr);
         EXPECT_FALSE(outcome.IsSuccess());
     }
 
     TEST_F(ActionManagerFixture, RegisterActionContext)
     {
-        auto outcome = m_actionManagerInterface->RegisterActionContext(m_widget, "o3de.context.test", "Test", "");
+        auto outcome =
+            m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, m_widget);
```

Comment:
nit/suggestion: I've found actually using `AzToolsFramework::ActionProperties{}` instead of `{}` makes it a lot easier to track down usages of `ActionProperties` - not super important but one of those things that in practice for maintenance can be useful